### PR TITLE
Replace humanmade with pressbooks coding standards

### PR DESCRIPTION
**Pull request by SteelWagstaff**

### CODE REVIEW

**Comment by SteelWagstaff** [comment]
@greatislander should we also update phpcs.ruleset.xml? Not sure I fully grasped what you were suggesting doing in our checkin today ...

**Review by greatislander — COMMENTED** [review]
Should also add:

- A composer script to run the standards check
- A Github action to run them on every PR / commit to dev or main

**Comment by SteelWagstaff** [comment]
@greatislander finally got around to doing this today. Not urgent, but if you're willing to take a look when you have a spare minute sometime in May?

File: phpcs.ruleset.xml [hunk 1]
```diff
@@ -10,19 +10,13 @@
   <arg value="p"/>
 
   <!-- Use HM Coding Standards -->
```
Comment (greatislander) [inline phpcs.ruleset.xml]:
```suggestion
  <!-- Use Pressbooks Coding Standards -->
```

**Comment by greatislander** [comment]
> @greatislander finally got around to doing this today. Not urgent, but if you're willing to take a look when you have a spare minute sometime in May?

Will do!

**Review by greatislander — APPROVED** [review]
This looks good!

**Pull request merged**